### DISCUSSION TITLE
How to setup TelegramBot

### DISCUSSION THREAD

**Discussion by Ryuspb** [Gozargah/Marzban#454] [Q&A]
Hello!
I have set  "TELEGRAM_API_TOKEN" and "TELEGRAM_ADMIN_ID" via .env! 
What to do next?

**Comment by IbragimGorch** [comment]
just put .env file in /opt/marzban/ folder 
and reboot

**Reply by Ryuspb** [reply]
I did it!

**Comment by IbragimGorch** [comment]
And enable it in WEB UI

**Reply by Ryuspb** [reply]
What section is this in? I could not find...

**Reply by IbragimGorch** [reply]
sorry, I was talking about another project (3x-ui) 
for Marzban it has to work when you put .env in correct folder 
try "marzban logs" it will say if something's wrong, I had problem with bot because of api_token (just revoked it and everything works fine)

**Reply by IbragimGorch** [reply]
try "cd /opt/marzban/ && ls -a1h" 
you have to see .env file

**Reply by Ryuspb** [reply]
.env file is in place, I added variables to it, but how can I see the log files? what file are they stored in?

**Reply by Ryuspb** [reply]
marzban logs doesn't throw any errors!

**Reply by Ryuspb** [reply]
I did it, thanks for the help!

**Comment by Ryuspb** [comment]
Maybe it will be useful to add step 3 like "don't forget to restart marzban (marzban restart)" via https://github.com/Gozargah/Marzban#telegram-bot section )
